src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/SetUpAction.kt

```diff
@@ -10,15 +10,8 @@ class SetUpAction(
 ) : Action {
     override fun run() {
         meter.measure(SET_UP) {
-            jira.configureRichTextEditor().disable()
-            val systemAdministrationPage = jira.administrate().system()
-            jira.accessAdmin().runWithAccess {
-                systemAdministrationPage
-                    .troubleshootingAndSupportTools()
-                    .instanceHealth()
-                    .notifications()
-                    .dontShowAny()
-            }
+            DisableRichTextEditor(jira, meter).run()
```

Comment:
Should `SetUpAction` still disable RTE?
If so, should it become deprecated?

Reply:
I'd leave the logic and `SetUpAction` deprecated - explaining why